inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -180,6 +196,8 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::ICNNNetwork &network, const st
         }
     }
 
+    OV_ITT_TASK_NEXT(taskChain, "make_shared<MKLDNNExecNetwork>");
+
```

Comment:
I  think we can clean this function form OV_ITT_TASK_CHAIN marker. 
The main point of interest are already covered by ITT instrumentation.

Reply:
I kept IE_OLD_ConstantFolding and last counter for network cloning.